nasr-charter.md

```diff
@@ -13,9 +13,9 @@ Path-agnostic traffic signing and encryption has been insofar the primary method
 * Unauthorized device root access, caused by physical tampering or penetration
 * Erroneous routing to unintended devices or areas, etc.
 
-Clients with high security and privacy requirements are not anymore satisfied with pure encryption-based data security measures, having no confidence of the security aspects of the underlying network elements. Clients now require proofs that data traverse through security-satisfying network elements (devices, links and services), avoiding any exposure to unqualifying elements. Clients would like to propose security requirements, such as paths composed of devices with the latest security updates, have passed integrity checks, or routes confined within specific geographical areas. Accordingly, operators should provide verifiable proofs to the clients for operational visualization, internal inspection and external auditing. 
+Customers with high security, privacy, and resiliency requirements are not any more satisfied with simple end-to-end encryption-based security measures, which does not allow assessing security and trustworthiness aspects of the underlying network elements. Emerging customers' demands include proofs that data traverse through network elements (devices, links and services) satisfying specific requirements, avoiding any exposure to unqualifying elements. Customers would like to propose path requirements, such as paths composed of devices with the latest security updates, have passed integrity checks, or routes confined within specific geographical areas. Accordingly, operators should provide verifiable proofs to the customers for operational visualization, internal inspection and external auditing. 
 
-RATS (Remote Attestation Procedures) working group has provided a framework and approaches to assess and establish the trustworthiness of a single device. Several individual submissions are also offering part of the solution to achieve NASR goal. However, a comprehensive framework that attests to a network-- meaning network-level security proofs and verification methods remains elusive.
+RATS (Remote Attestation Procedures) working group has provided a framework and approaches to assess and establish the trustworthiness of a single device, hence offering an initial building block. However, a comprehensive framework that attests to a network -- meaning network-level elements' trustworthiness proofs and verification methods remains elusive.
```

Comment:
```suggestion
The RATS (Remote ATtestation procedureS) working group has provided a framework and approaches to assess and establish the trustworthiness of a single device, hence offering an initial building block. 
However, a comprehensive framework that attests to a network -- meaning network-level elements' trustworthiness proofs and verification methods remains elusive.
```

Reply:
I don't think we want to say, "remains elusive" --- that kinda implies it's unreachable to us as well.